src/main/java/io/aiven/kafka/connect/opensearch/OpensearchSinkConnectorConfig.java

```diff
@@ -145,13 +145,18 @@ public class OpensearchSinkConnectorConfig extends AbstractConfig {
     public static final String BEHAVIOR_ON_MALFORMED_DOCS_CONFIG = "behavior.on.malformed.documents";
     private static final String BEHAVIOR_ON_MALFORMED_DOCS_DOC = "How to handle records that "
             + "Opensearch rejects due to some malformation of the document itself, such as an index"
-            + " mapping conflict or a field name containing illegal characters. Valid options are "
-            + "'ignore', 'warn', and 'fail'.";
+            + " mapping conflict or a field name containing illegal characters. Valid options are: "
+            + "'ignore' - do not index the record, 'warn' - log a warning message and do not index the record, "
+            + "'report' - report to errant record reporter and do not index the record, 'fail' - fail the task.";
     
     public static final String BEHAVIOR_ON_VERSION_CONFLICT_CONFIG = "behavior.on.version.conflict";
     private static final String BEHAVIOR_ON_VERSION_CONFLICT_DOC = "How to handle records that "
             + "Opensearch rejects due to version conflicts (if optimistic locking mechanism has been"
-            + "activated). Valid options are 'ignore', 'warn', and 'fail'.";
+            + "activated). Valid options are: 'ignore' - ignore and keep the existing record, "
+            + "'warn' - log a warning message and keep the existing record, 'report' - report to errant record reporter"
+            + " and keep the existing record, 'fail' - fail the task.";
+
+    public static final String DLQ_TOPIC_NAME_CONFIG =  "errors.deadletterqueue.topic.name";
```

Comment:
I think we can just remove this field completely, since it's not used anywhere in the main codebase, and is not actually necessary for testing (see comments in `BulkProcessorTest`).